src/on-create-node.ts

```diff
@@ -19,10 +19,11 @@ export type GatsbyPluginArgs = {
 };
 
 export const onCreateNode = (
-  { node, getNodesByType }: GatsbyPluginArgs,
+  { node, getNodesByType, actions }: GatsbyPluginArgs,
```

Comment:
@AyronK actions type definition is missing, should be: 


```
actions: {
    createNodeField: (args: {
      node: MarkdownNode;
      name: string;
      value: any;
    }) => void;
  };
```

Reply:
Fixed